src/client/post/postsReducer.js

```diff
@@ -57,16 +57,26 @@ const posts = (state = initialState, action) => {
     case feedTypes.GET_USER_FEED_CONTENT_SUCCESS:
     case feedTypes.GET_MORE_USER_FEED_CONTENT_SUCCESS:
     case bookmarksActions.GET_BOOKMARKS_SUCCESS: {
-      const postsTemp = {};
+      const list = {
+        ...state.list,
+      };
+      const postsStates = {
+        ...state.postsStates,
+      };
+
       action.payload.postsData.forEach((post) => {
```

Comment:
I think I remember seeing somewhere that LoDash's _.each is faster for some browsers than the normal native `.forEach` 

https://stackoverflow.com/a/19206409/2884655

Your call whether to use it or not :)